tools/oecert/host/evidence.cpp

```diff
@@ -131,30 +131,30 @@ void parse_certificate_extension(const uint8_t* data, size_t data_len)
     printf(
         "\n    parsed qe certificate extension (%s) {\n",
         SGX_EXTENSION_OID_STR);
-    printf("        ppid (hex): ");
+    printf("      ppid (hex): ");
```

Comment:
All the changes in this file look unrelated to the description of this PR.
https://github.com/openenclave/openenclave/blob/master/docs/Community/Contributing.md says:

> * **DON'T** mix independent, unrelated changes in one PR. Separate real project/test code changes from larger code formatting/dead code removal changes. Separate unrelated fixes into separate PRs, especially if they are in different libraries.

Reply:
I'll put them in another PR.